050-AIS-APIManagementwithFunctions/Coach/README.md

```diff
@@ -0,0 +1,85 @@
+
+## Coach's Guides
+<!-- REMOVE_ME ${challengesSection} (remove this from your MD files if you are writing them manually, this is for the automation script) REMOVE_ME -->
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+-  Challenge 0: **[Prepare your Development Environment](Solution-00.md)**
+   - Get yourself ready to build your integration solution
+-  Challenge 1: **[Provision an Integration Environment](Solution-01.md)**
+   - Create a bicep template that will provision a baseline integration environment.
+-  Challenge 2: **[Deploy your Integration Environment](Solution-02.md)**
+   - Create a CI/CD pipeline to do automated deployment of your integration environment.
+-  Challenge 3: **[Create Backend APIs](Solution-03.md)**
+   - Create backend APIs
+-  Challenge 4: **[Secure Backend APIs](Solution-04.md)**
+   - Securing backend APIs by configuring them in the VNET or by using OAuth 2.0 authorization
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Coach Prerequisites 
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the \`/Student/Resources\` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.  
+
+### Additional Coach Prerequisites 
+- Visual Studio Code 1.63 or higher with Bicep extension
+- Azure CLI 2.20.0 or higher
+- Azure DevOps project or GitHub project
+
+## Azure Requirements
+
+This hack requires students to have access to an Azure subscription where they can create and consume Azure resources. These Azure requirements should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students.
+
+
+## Suggested Hack Agenda 
+- Day 1
+	- Challenge 0 (1 hour)
+	- Challenge 1 (2 hours)
+	- Challenge 2 (1 hour)
+- Day 2
+	- Challenge 2 (1 hour)
+ 	- Challenge 3 (1 hour)
+ 	- Challenge 4 (1 hour)
+
+## Repository Contents
+
+*The default files & folders are listed below. You may add to this if you want to specify what is in additional sub-folders you may add.*
+
+- \`./Coach\`
```

Comment:
Due to the fact that template was used manually, the monospaced characters here needed to have escape characters ("\") in front of the backticks so it would render appropriately.

Please remove the "\" escape characters from all of the folder names here.